packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md

```diff
@@ -0,0 +1,50 @@
+---
+id: Timestamp
+section: components
+---
+
+## Elements
+
+<img src="./img/timestamp-with-tooltip-underline.png" alt="Timestamp with tooltip underline"/>
+<img src="./img/timestamp-tooltip-popover.png" alt="Timestamp with tooltip popover"/>
+
+1. **Day:** Spelled out or abbreviated format. [See Numerics guidelines for more information](https://www.patternfly.org/v4/ux-writing/numerics)
+2. **Date:** Can be spelled out, abbreviated form, or digit format. [See Numerics guidelines for more information](https://www.patternfly.org/v4/ux-writing/numerics)
+3. **Time:** Should be displayed in 12-hour or 24-hour ((HH:MM:SS). [See Numerics guidelines for more information](https://www.patternfly.org/v4/ux-writing/numerics)
+4. **UTC/Timezone:** Should always be displayed last, in shorthand form and be in 24-hour format.
+5. **Tooltip:** Dotted line displayed if there is tooltip available.
+
+
+## Usage
+### Use timestamp when:
+You want to represent the exact date and time that an event occurred When users can go back and make use of past content.
+
+### Best practices
+#### For timestamp:
+- Depending on the context, show either date or time, or both date and time.
+- Locale should be appropriately set so that users see dates and times in a familiar format.
+- Timestamp at minimum should include date, time, and timezone (shorthand format).
+- Every element does not need to be used when implementing a timestamp.
+#### For tooltip:
+- Not required but recommended to provide more precise time for relative times. Tooltips should show timestamps in UTC time.
+- Recommend displaying time in 24-hour format
+- Displays time in UTC (Coordinated Universal Time).
+- Should be formatted the same as the timestamp. 
+
+See UX [numerics guidelines](https://www.patternfly.org/v4/ux-writing/numerics) for more information about relative time best practices.
+
+## Variations
+### 12-hour format
+In the 12-hour format use AM to refer to a time of day before noon, and PM to speak of a time between noon and midnight.
+
+<img src="./img/timestamp-am-pm-format.png" alt="Timestamp AM/PM format"/>
+
+### 24-hour format
+In the 24-hour clock format, each day runs from midnight to midnight and is divided into 24 hours, starting at 1 a.m. 
```

Comment:
Doesn't it start at midnight, with 00:00? I think it is confusing saying it runs from midnight to midnight and then say it starts at 1. What about " each day runs from midnight to midnight and is divided into 24 hours, beginning with 00:00 AM at midnight."

Reply:
sounds good @wolfeallison. Midnight is a 0 & 24 hour which can be confusing as well, which is why I chose to use this specific definition.  I'm fine with the definition you've written out, will update.